backend/src/main/java/com/redhat/cloud/notifications/auth/principal/rhid/RhServiceAccountIdentity.java

```diff
@@ -0,0 +1,41 @@
+package com.redhat.cloud.notifications.auth.principal.rhid;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class RhServiceAccountIdentity extends RhIdentity {
```

Comment:
Does `ConsoleIdentityProvider` work out of the box with this class?
https://github.com/RedHatInsights/notifications-backend/blob/ddfb7dffa9979ac027f8ac6d2a62d0bf56a048d0/backend/src/main/java/com/redhat/cloud/notifications/auth/ConsoleIdentityProvider.java#L122-L123

Reply:
Yes it have to according the ADR